torch/_export/serde/schema.py

```diff
@@ -200,6 +217,7 @@ class Graph:
     nodes: List[Node]
     tensor_values: Dict[str, TensorMeta]
     sym_int_values: Dict[str, SymInt]
+    sym_float_values: Dict[str, SymFloat]
```

Comment:
maybe modify this to `sym_float_values: Dict[str, SymFloat] = field(default_factory=list)`, as well as the other values
to avoid a major schema bump

@yiming0416 for serialization BC, is this appropriate?

Reply:
Do you mean just to add `= field(default_factory=list)` ?

Reply:
Yeah. I think you'll have to add default values here, and likely also upgrade the schema version, to pass the schema compatibility test + avoid a major version bump.

Reply:
Right, sorry. Currently the schema version is 8.1 (defined top of the file). When we add new fields to existing schema dataclasses, we would like them to have default values (so if we deserialize an older program, we know what value to provide when that field is missing). Providing these default values allows this PR to only bump the minor version (8.1 -> 8.2), which is desirable for internal testing purposes. Without the default values, we have to bump the major version (-> 9.1), which is not so desirable, because now we're declaring 8.1 serialized programs incompatible. Also because it's very avoidable in this case - empty dicts and empty strings.

I think in this case `sym_float_values` is the only one that needs a default value, since `as_sym_float` & `as_sym_floats` are union types? But I could be wrong, will wait to see CI.

Reply:
We have this for the schema from the CI https://github.com/pytorch/pytorch/actions/runs/11919082785/job/33220062570?pr=136364

Reply:
The CI is done so let me know what we need to do.